### ISSUE TITLE
Reverting resolution after 120 seconds

### ISSUE THREAD

**Issue by seanstardev** [Nonary/ResolutionAutomation#44]
Thank you for this great software.

Only issue I'm having is that the original resolution does not kick back in after 120 seconds of the stream being paused.

I edited the scripts and changed it to take effect after 4 seconds (to be able to test more quickly). But no luck.

Is this definitely working? Is the IsCurrentlyStreaming variable being set correctly?

Many thanks :)

**Comment by Nonary** [comment]
Yes it should be working, at least for the sunshine version of the script. In fact people often have the opposite issue, wanting to extend that period instead of reduce it. It can be changed in the settings file

**Comment by seanstardev** [comment]
> Yes it should be working, at least for the sunshine version of the script. In fact people often have the opposite issue, wanting to extend that period instead of reduce it. It can be changed in the script itself, search for 120 and replace it

Thank you very much for the response. 

I am using Sunshine and I did indeed replace the '120' with '4':

```
else {
                if (((Get-Date) - $lastStreamed).TotalSeconds -gt 4) {
                    Write-Output "Ending the stream script"
                    New-Event -SourceIdentifier ResolutionMatcher -MessageData "End"
                    break;
                }
    
            }
```

When the 'Stream Paused' alert shows for Sunshine, I am expecting the resolution to revert after 4 seconds. Would this be the behaviour I should expect, or am I missing something?

Explicitly closing the session through Moonlight immediately prompts the host's resolution to revert, which is fantastic, so no dramas there. 

The Sunshine config edit looks like this when install is successful:

global_prep_cmd = [{"do":"powershell.exe -executionpolicy bypass -WindowStyle Hidden -file \"Z:\\Apps\\!3rd Party\\ResolutionAutomation\\ResolutionMatcher.ps1\"","elevated":"false","undo":"powershell.exe -executionpolicy bypass -WindowStyle Hidden -file \"Z:\\Apps\\!3rd Party\\ResolutionAutomation\\ResolutionMatcher-Functions.ps1\" True"}]

But nothing in the block with this comment is being executed *I put a load of debug log text in there, and nothing is printed):

# Asynchronously start the ResolutionMatcher, so we can use a named pipe to terminate it.

**Comment by seanstardev** [comment]
Is there not a confusion between IsCurrentlyStreaming and UserIsStreaming() between the two ps files, in the latest release? 

Really confused by the nested zip in the source file as well lol. Will keep looking XD

**Comment by Nonary** [comment]
Fixed in latest release
